configuration/concepts/lab_results_form.csv

```diff
@@ -0,0 +1,334 @@
+59c83cc7-0e28-44ab-a8bd-2ebe20a57a50,,,Crachants,Crachants,Crachants,Crachants,LabTest,Text,,
+08036341-6ff3-40b4-9a52-5a01beb4e673,,,Crachants Date de résultat,Crachants Test Date,Date de résultat,Test Date,Misc,Date,,
+13ec5ffe-9874-428d-919d-15c29bbc9801,,,LCR,CSF parasitology,LCR,CSF parasitology,LabTest,Text,,
+0d9f270d-b2a5-42cf-ac8e-d5d631fc58e3,,,LCR Date de résultat,CSF parasitology Test Date,Date de résultat,Test Date,Misc,Date,,
+3b08b387-a574-49bb-975e-d2e0c5698606,,,Coprology,Coprology,Coprology,Coprology,LabTest,Text,,
+da7d03c3-130c-40ab-985e-78c4c87d8248,,,Coprology Date de résultat,Coprology Test Date,Date de résultat,Test Date,Misc,Date,,
+23ba1e3f-78d4-4f0c-9074-bcf72e02e29b,,,Test de Diagnostic Rapide du Paludisme,Rapid Diagnosis Malaria Test,Test de Diagnostic Rapide du Paludisme,Rapid Diagnosis Malaria Test,LabTest,Text,,
+51b23d51-f3f5-474e-907e-254fbe62d429,,,Test de Diagnostic Rapide du Paludisme Date de résultat,Rapid Diagnosis Malaria Test Test Date,Date de résultat,Test Date,Misc,Date,,
+18469f86-6709-4449-b965-590b86e45cae,,,Test du Paludisme (GE),Malaria Test (GE),Test du Paludisme (GE),Malaria Test (GE),LabTest,Text,,
+a4d19a14-c6bc-4323-be6d-06792b47487e,,,Test du Paludisme (GE) Date de résultat,Malaria Test (GE) Test Date,Date de résultat,Test Date,Misc,Date,,
+d236def5-5ac9-440e-851e-32d76e82193b,,,Recherche des Micro Filaires,Recherche des Micro Filaires,Recherche des Micro Filaires,Recherche des Micro Filaires,LabTest,Text,,
+94659510-1ffc-4982-b3b7-9acee62dca84,,,Recherche des Micro Filaires Date de résultat,Recherche des Micro Filaires Test Date,Date de résultat,Test Date,Misc,Date,,
+baf5d5ac-47a0-4733-b58c-dfc8ef64530f,,,Skin snip,Skin snip,Skin snip,Skin snip,LabTest,Text,,
+4d4d0e97-af80-4e58-a63d-6739d133b4ec,,,Skin snip Date de résultat,Skin snip Test Date,Date de résultat,Test Date,Misc,Date,,
+7479640d-b96d-442c-be05-542bcd009a65,Crachants;Crachants Test Date;CSF parasitology;CSF parasitology Test Date;Coprology;Coprology Test Date;Rapid Diagnosis Malaria Test;Rapid Diagnosis Malaria Test Test Date;Malaria Test (GE);Malaria Test (GE) Test Date;Recherche des Micro Filaires;Recherche des Micro Filaires Test Date;Skin snip;Skin snip Test Date,,Département de Parasitologie,Parasitology Dept,Département de Parasitologie,Parasitology Dept,Misc,N/A,,
+3b5b8d72-2f86-48fc-9b81-0be908ed392c,Bacteriology Dept;Haematology Dept;Biochemistry Dept;Parasitology Dept,,Résultats de laboratoire - ajoutés manuellement,Lab results - add manually,Résultats de laboratoire - ajoutés manuellement,Lab results - add manually,Misc,N/A,,
```

Comment:
@lauravignoli Please use full caps for this concept's name

Reply:
Fixed